### ISSUE TITLE
Cannot pipe data from other events

### ISSUE THREAD

**Issue by pbchase** [ctsit/redcap_chart_field#3]
In my testing with Chart, I could not pipe data from other events.  Using these events

```
unique_event_name
--
event_1_arm_1
event_2_arm_1
event_3_arm_1
event_4_arm_1
```

Configuration like this

```
data: ['[event_1_arm_1][foo]', '[event_2_arm_1][foo]', '[event_3_arm_1][foo]', '[foo]' ],
```

Produced javascript that looked like this

```
data: ['[event_1_arm_1]1', '[event_2_arm_1]1', '[event_3_arm_1]1', '1' ]
```

When _foo_ had a value of 1 on the current event.

**Comment by tbembersimeao** [comment]
Nice catch! Created a [PR](https://github.com/ctsit/redcap_chart_field/pull/5) to fix this.

**Comment by pbchase** [comment]
Fixed by #5
